packages/unmock-cli/src/commands/list.ts

```diff
@@ -15,37 +16,44 @@ interface IStringMap {
   [key: string]: string[];
 }
 
-export const list = () => {
-  const logger = new WinstonLogger();
-  logger.log("Fetching mocks and relevant tests...");
-  glob(
-    `${process.cwd()}/**/${DEFAULT_SNAPSHOTS_FOLDER}/**/*${SNAP_SUFFIX}`,
-    { dot: true },
-    (e, matches) => {
-      if (e) {
-        throw e;
-      }
-      matches.forEach((match) => {
-        logger.log(path.basename(match, SNAP_SUFFIX));
-        const snapshots = JSON.parse(readFileSync(match, "utf8"));
-        const tests: IStringMap = Object.keys(snapshots).reduce(
-          (obj: IStringMap, name) => {
-            const key = removeTrailingNumber(name);
-            (obj[key] = obj[key] || []).push(name);
-            return obj;
-          },
-          {},
-        );
-        Object.keys(tests).forEach((test) => {
-          logger.log(`  ${test}`);
-          tests[test].forEach((call) => {
-            logger.log(`    hash: ${snapshots[call].hash}`);
-            logger.log(`      method: ${snapshots[call].method}`);
-            logger.log(`      host: ${snapshots[call].host}`);
-            logger.log(`      path: ${snapshots[call].path}`);
+export const listInternal = (logger: ILogger): Promise<undefined> => {
+  return new Promise((resolve, reject) => {
```

Comment:
Do we plan on chaining commands to this?

Reply:
And since I see you've used `JSON.stringify` in `curl` - should we also maintain that implementation here? (build an object with the list content and let JSON format it?)